packages/patternfly-4/react-core/src/components/Select/CheckboxSelectOption.tsx

```diff
@@ -91,7 +91,7 @@ export class CheckboxSelectOption extends React.Component<CheckboxSelectOptionPr
               checked={isChecked || false}
               disabled={isDisabled}
             />
-            <span className={css(checkStyles.checkLabel, isDisabled && styles.modifiers.disabled)}>{value}</span>
+            <span className={css(checkStyles.checkLabel, isDisabled && styles.modifiers.disabled)}>{children && children || value}</span>
```

Comment:
Isn't this equivalent to `{children}{value}`, which makes more sense?

Reply:
@redallen I don't think so.  value is always defined.  The way I am understanding the code is children will rendered if it is defined, otherwise value will be rendered. We never want to render both as `{children}{value}` would do if children is defined. Would updating the prop description help?

Reply:
Ah, in that case it should just be `{children || value}`. `{children && children}` is equivalent to `{children}`.